src/components/upload.css

```diff
@@ -0,0 +1,34 @@
+@font-face {
+    font-family: 'Inter';
+    src: local('Inter'), url(../fonts/Inter/Inter-VariableFont_slnt,wght.ttf) format('truetype');
+    font-weight: 100 1000;
+	font-stretch: 25% 151%
+  }
+
+.upload-btn {
+  margin: auto;
+  margin-top: 8px;
+  width: 200px;
+  font-size: 15px;
+  padding: 10px 5px;
+  border-radius: 20px;
```

Comment:
You should use `rem` units rather than pixel units. Pixel units won't support different screen resolutions e.g. someone with a 4k monitor

Reply:
ooooooooo ok